lib/run.rb

```diff
@@ -1,34 +1,101 @@
 require_relative '../config/environment'
 CLEAR = "\e[H\e[2J"
+PROMPT = TTY::Prompt.new
+OPTIONS =  %w(Price_Check Compare_Cryptos My_Portfolio Close)
+PING_URL = "https://api.coingecko.com/api/v3/ping"
```

Comment:
Could this live in it's own class where it's domain is just to do with connecting with the poke api?